rollup.config.js

```diff
@@ -25,6 +25,15 @@ import inject from '@rollup/plugin-inject'
 const production = !process.env.ROLLUP_WATCH
 const sourcemap = process.env.SOURCE_MAP === 'true'
 const { version } = require('./package.json')
+const compilationTimestamp = new Date().getTime()
+
+const config = {
+  requirejs: {},
+  cdn: process.env.CDN === 'true'
```

Comment:
Would've loved a changelog item or docs reference for this option, but can be added later as well:)